src/modules/launcher/Plugins/Microsoft.Plugin.WindowWalker/Properties/Resources.resx

```diff
@@ -190,4 +190,16 @@
   <data name="wox_plugin_windowwalker_SettingKillProcessTree" xml:space="preserve">
     <value>Kill process and its child processes</value>
   </data>
+  <data name="wox_plugin_windowwalker_SettingExplorerSettingInfo_Description" xml:space="preserve">
+    <value>This message is only shown, when searching with direct activation command.</value>
+  </data>
+  <data name="wox_plugin_windowwalker_SettingKillProcessTree_Description" xml:space="preserve">
+    <value>Be careful when activating this.</value>
+  </data>
+  <data name="wox_plugin_windowwalker_SettingOpenAfterKillAndClose_Description" xml:space="preserve">
+    <value>This feature isn't working, if the kill process confirmation is enabled.</value>
```

Comment:
Shouldn't we disable this if kill process confirmation is enabled? (Ideally, but I can imagine that might be challenging).

Suggested tweak: "This feature won't work if the kill process confirmation is enabled"

Reply:
This reason of this is that wox hides if an other window has focus (in this case the message box). We have an issue to improve the closing behavior and I think this will be fixed then too.

I don't think we should invest the time to disable this. I think the warning is enough. I will update the text in the next days.